defi/src/protocols/data.ts

```diff
@@ -1317,7 +1317,7 @@ export default [
     logo: `${baseIconsUrl}/synlev.jpg`,
     audits: "0",
     audit_note: null,
-    gecko_id: "synlev",
+    gecko_id: null,
```

Comment:
did these assets get delisted?

Reply:
I don't know what happened but the links have been disabled on coingecko, twitter, discord and telegram.